src/main/liquibase/liquibase-ciel-data.xml

```diff
@@ -5,33 +5,31 @@
     http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
 
 	<changeSet id="20120315-1000" author="standalone" dbms="mysql">
-		<comment>Preparing schema for adding mvp data</comment>
```

Comment:
Before this line, please add the line `<validChecksum>3:dc136a76a36273b5cdb2f47b0eb24522</validChecksum>`